src/System.Text.Encodings.Web/src/project.lock.json

```diff
@@ -1,18 +1,15 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
Yea it shouldn't have the lock file checked in. We're actually going to be removing this property.